src/views/listview/list-view-directive.js

```diff
@@ -238,6 +242,18 @@
           },
         ];
 
+        $scope.getMenuClass = function (item) {
+          var menuClass = "";
+          if (item.name === "Jim Beam") {
```

Comment:
Maybe a slightly less risque name here?  Not sure if we have public standards for documentation or if I'm just being overly cautious...